templates/main/base.html

```diff
@@ -37,5 +37,12 @@
     {% endblock %}
 {% endif %}
 {% block content %}{% endblock %}
+<footer>
+<div class="uk-section uk-section-secondary uk-container-medium uk-text-meta">
+    <div class="uk-text-center">
+        <p>Developed collaboratively on <a href="https://github.com/discord-python/site">Github</a> • 2018</p>
```

Comment:
The • symbol should not be a literal, but instead should use the equivalent html entity.